tests/gis_tests/test_postgis_extension.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+
+from django.db import connection
+from django.test import TestCase
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', "PostgreSQL specific tests")
+class TestsPostGISCreateExtension(TestCase):
+
+    def setUp(self):
+
```

Comment:
Generally you have too many blank lines. Please try to follow the style of existing tests, and when in doubt, don't include a blank line.

Reply:
I will obey and read more about the coding style and fix it. Thanks